src/css-grids/cssgrids.css

```diff
@@ -0,0 +1,48 @@
+.main-container {
+    background: url("flag.jpg");
```

Comment:
change this to `url("./flag.jpg")`
while your way works, it's good practice to add the ./ before a file that is a sibling file